public/app/features/inspector/InspectErrorTab.tsx

```diff
@@ -18,5 +18,20 @@ export const InspectErrorTab: React.FC<InspectErrorTabProps> = ({ error }) => {
       </>
     );
   }
-  return <div>{error.message}</div>;
+  if (error.message) {
+    try {
+      const [msg, json] = error.message.split(/\{(.+)/);
+      const jsonError = JSON.parse('{' + json);
+      const title = msg && msg !== '' ? <h3>{msg}</h3> : null;
```

Comment:
from the screenshots, it seems that the `<h3>` is doing nothing, right?

Reply:
If there are both a message and a json in the error message, the message is displayed as h3 and the json formatted.
You can see it in the second inspector screenshot ("400 Bad Request...") : 
<img width="300" alt="msgJson" src="https://user-images.githubusercontent.com/42030685/154453914-9375af74-24a7-4eb1-9672-b28214383cd6.png">

Reply:
ah, gotcha